src/main/resources/substitutionTemplates/conversational-search-with-cohere-model-template.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "${{template.name}}",
+  "description": "${{template.description}}",
+  "use_case": "SEMANTIC_SEARCH",
+  "version": {
+    "template": "1.0.0",
+    "compatibility": [
+      "2.12.0",
+      "3.0.0"
+    ]
+  },
```

Comment:
We should make clear in the documentation these template schemas are subject to change. For example, once workflow versioning is available, all of these templates will need to be updated, and these existing ones won't be compatible.

Reply:
will add this to documentation